tests/unittests/OperatorTest.cpp

```diff
@@ -26,22 +26,44 @@
 #include "llvm/Support/raw_ostream.h"
 
 using namespace glow;
+
 class Operator : public ::testing::TestWithParam<BackendKind> {
 public:
   Operator() : mod_(EE_.getModule()) { F_ = mod_.createFunction("main"); }
 
   ~Operator() override { mod_.clear(); }
 
 protected:
+  bool isEnabledBackend(const std::set<BackendKind>& enabledBackends) {
```

Comment:
CI is failing on this line because of clang-format

Reply:
clang-format applied.